lib/lighthouse/benefits_documents/configuration.rb

```diff
@@ -85,9 +86,13 @@ def generate_upload_body(document_data, file_body)
       File.write(fn, data.to_json)
       payload[:parameters] = Faraday::UploadIO.new(fn, 'application/json')
 
+      file_type = MimeMagic.by_path(document_data[:file_name]).subtype
+      allowed_filetype(file_type)
```

Comment:
Sounds good. updated. 

For the exception handling. since this happens asynchronously, this error won't reach any controllers. The app will just know if the job got kicked off or not. not ideal for knowing if their document got uploaded properly but this was how it was done for EVSS. @Tonksthebear wrote that bit, I assume there's a functional reason for it like time it takes for uploads to complete.